test/dummy_test.go

```diff
@@ -375,3 +393,16 @@ func TestDummySequencer_VerifyBatch_NotFound(t *testing.T) {
 	assert.NoError(t, err)
 	assert.False(t, verifyResp.Status)
 }
+
+// GenerateSecureRandomBytes generates cryptographically secure random bytes of the given length.
+func GenerateSecureRandomBytes(length int) ([]byte, error) {
```

Comment:
nit: imo for test helpers like this, you should just panic on error to save all the error checking in the tests. Since errors here are developer errors and the test would be expected to fail until fixed.